uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -87,6 +92,7 @@ public static UiObject obtainUiObjectWithResourceIdAndEnabledFlag(@NonNull final
      */
     @NonNull
     public static UiObject obtainEnabledUiObjectWithExactText(@NonNull final String text) {
+        Logger.i(TAG, "Obtain an instance of an enabled UiObject for the given text..");
```

Comment:
let's also log the text in the log message

Reply:
added.